redbot/core/core_commands.py

```diff
@@ -1439,15 +1439,31 @@ async def helpset_showhidden(self, ctx: commands.Context, show_hidden: bool = No
             show_hidden = not await ctx.bot._config.help.show_hidden()
         await ctx.bot._config.help.show_hidden.set(show_hidden)
         if show_hidden:
-            await ctx.send(_("Help will not filter hidden commands"))
+            await ctx.send(_("Help will not filter hidden commands."))
         else:
             await ctx.send(_("Help will filter hidden commands."))
 
+    @helpset.command(name="showaliases")
+    async def helpset_showaliases(self, ctx: commands.Context, show_aliases: bool = None):
+        """
+        This allows the help command to show existing commands aliases if there is any.
+
+        This defaults to True.
+        Using this without a setting will toggle.
+        """
+        if show_aliases is None:
+            show_aliases = not await ctx.bot._config.help.show_aliases()
+        await ctx.bot._config.help.show_aliases.set(show_aliases)
+        if show_aliases:
+            await ctx.send(_("Help will not show commands aliases."))
+        else:
+            await ctx.send(_("Help will show commands aliases."))
```

Comment:
These two strings should be the other way around.
```suggestion
        if show_aliases:
            await ctx.send(_("Help will show commands aliases."))
        else:
            await ctx.send(_("Help will not show commands aliases."))
```